Project/Assets/Editor/Annotations/AnnotationInteractionEditor.cs

```diff
@@ -0,0 +1,147 @@
+﻿using UnityEngine;
+using UnityEditor;
+using System.IO;
+
+
+[CustomEditor(typeof(AnnotationInteraction))]
+public class AnnotationInteractionEditor : Editor {
+
+    bool showRichText = true;
+    AnnotationInteraction annotation;
+
+
+    public override void OnInspectorGUI()
+    {
+        annotation = (AnnotationInteraction)target;
+
+        string[] options = new string[] { "Import Text File", "Write in Inspector" };
+        annotation.selected = EditorGUILayout.Popup("Import Method", annotation.selected, options);
+
+        //Processing if importing a text file
+        if (annotation.selected == 0)
+        {
+            DisplayImportFile();
+
+        } else //Process text written in inspector
+        {
+            DisplayWriteInInspector();
+        }
+    }
+
+    /// <summary>
+    /// Does the processing for displaying Inspector options when choosing to Import a file
+    /// </summary>
+    void DisplayImportFile()
+    {
+        GUILayout.BeginHorizontal();
+        GUILayout.Label("Annotation File:");
+        ImportFileButton();
+        GUILayout.EndHorizontal();
+
+        annotation.includeImages = EditorGUILayout.Toggle("Include Images:", annotation.includeImages);
+        if (annotation.includeImages)
+        {
+            GUILayout.Label("Do the images share a directory with the text file?");
+            annotation.sharedFile = EditorGUILayout.Toggle("Shared Directory:", annotation.sharedFile);
+
```

Comment:
This is a great way to go about solving the problem of tracking down the images directory.